### ISSUE TITLE
[Sync] 

### ISSUE THREAD

**Issue by wtrocki** [aerogear/aerogear-js-sdk#291]
## Motivation

Use more efficient storage solution that do not perform JSON.stringify etc.
Proposed option: https://github.com/localForage/localForage

**Comment by wtrocki** [comment]
ping @DaraHayes @StephenCoady

**Comment by darahayes** [comment]
Yeah according to the upstream documentation localForage is 'supported out of the box' so it would be worth testing out as a drop in replacement. https://github.com/apollographql/apollo-cache-persist#storage-providers

**Comment by wtrocki** [comment]
Yes. I have noticed huge performance gains on application startup even on simple showcase dataset. Rehydrating localstorage takes long time when is persisted as string (JSON.strinfigy). We need that as default option (it is simple to pass that into the showcase application but I have been using this for some time now and it seems that it will be seamless transition.

**Comment by wtrocki** [comment]
TODO! Migrate this to jira.
